### DISCUSSION TITLE
Get-PnPListItemVersion Returns nothing

### DISCUSSION THREAD

**Discussion by Mr-Clrscr** [pnp/powershell#3576] [Q&A]
Hello Community,

I'm having below snippet which will loop through each item version history and checks for specific string is present in title column or not. 

And I've an item with 3 versions and the child loop is looping thrice as expected. However, I'm not getting any value in either $version.FieldValues.Title or $version.FieldValues. Am I missing something here ?

`foreach ($item in $items) {
    Write-Host "ID: $($item.Id)"
    $itemVersions = Get-PnPListItemVersion -List $listName -Identity $item.Id
    foreach ($version in $itemVersions) {
        Write-Host "$version"
        if ($version.FieldValues.Title -eq $searchString) {
            Write-Host "ID: $($item.Id)"
            $found = $true
            break
        }
    }
}`

**Comment by AndersRask** [comment]
If i use this code i get the title for each version just fine. 

```
foreach ($item in $items) {
   Write-Host ("ID: {0}" -f $item.Id)
   $itemVersions = Get-PnPListItemVersion -List $listName -Identity $item.Id
   foreach ($version in $itemVersions) {
   Write-Host ("{0} [{1}]" -f $version.VersionLabel, $version.Values.Title)
 }
}
ID: 16
8.0 [This has      a     lot of spaces sure has]
7.0 [This has      a     lot of spaces sure had]
6.0 [This has      a     lot of spaces sure had]
5.0 [This has      a     lot of spaces sure had]
4.0 [This has      a     lot of spaces]
3.0 [This has      a     lot of spaces]
2.0 [This has      a     lot of spaces]
1.0 []
```

If you are searching for a substring in the title you should use either regex -match or use -like
```

foreach ($item in $items) {
  Write-Host ("ID: {0}" -f $item.Id)
  $itemVersions = Get-PnPListItemVersion -List $listName -Identity $item.Id
  foreach ($version in $itemVersions) {
  Write-Host ("{0} [{1}]" -f $version.VersionLabel, $version.Values.Title)
  if ( $version.Values.Title -like "*had*" ) { break }
 } }
ID: 16
8.0 [This has      a     lot of spaces sure has]
7.0 [This has      a     lot of spaces sure had]
```
